Work with text/src/com/epum/primisen/task_02/parse_text/paragraph/ParagraphLogic.java

```diff
@@ -0,0 +1,26 @@
+package com.epum.primisen.task_02.parse_text.paragraph;
+
+import com.epum.primisen.task_02.parse_text.sentence.Sentence;
+import com.epum.primisen.task_02.parse_text.utill.Regex;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class ParagraphLogic {
```

Comment:
For this and all other ...Logic classes make private constructor. Utility classes should not have public constructors